package.json

```diff
@@ -305,9 +315,10 @@
         "vscode:prepublish": "npm run recompile",
         "bundleDeps": "node ./build-scripts/bundleDeps.js",
         "clean": "node ./build-scripts/clean.js out",
+        "createWelcomePage": "node ./build-scripts/createWelcomePage.js",
         "reset": "node ./build-scripts/clean.js out node_modules && npm install",
         "copyNonCodeFiles": "node ./build-scripts/copyNonCodeFiles.js",
-        "compile": "tsc -p ./ && npm run lint && npm run bundleDeps && npm run copyNonCodeFiles",
+        "compile": "tsc -p ./ && npm run lint && npm run bundleDeps && npm run copyNonCodeFiles && npm run createWelcomePage",
```

Comment:
Can you add a script (something like `prepare` or `prebuild`) that runs `bundleDeps`, `copyNonCodeFiles` and `createWelcomePage`, then run the new script from both `compile` and `watch`? Otherwise, running `npm i` and `npm watch` directly after cloning will result in a broken build.

Reply:
✔️